src/rules/__tests__/no-try-expect.test.js

```diff
@@ -0,0 +1,66 @@
+import { RuleTester } from 'eslint';
+import rule from '../no-try-expect';
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
```

Comment:
can you use the default parser, but set `ecmaVersion` to something that supports `async-await` (which I assume is why you needed a different parser)